aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -31,8 +32,8 @@
 RETRIEVE_COMMAND = 'retrieve'
 KILL_COMMAND = 'kill'
 
-TRANSPORT_TASK_RETRY_INITIAL_INTERVAL = 20
-TRANSPORT_TASK_MAXIMUM_ATTEMTPS = 5
+TRANSPORT_TASK_RETRY_INITIAL_INTERVAL = get_config().get_option('transport.task_retry_initial_interval')
+TRANSPORT_TASK_MAXIMUM_ATTEMPTS = get_config().get_option('transport.task_maximum_attempts')
```

Comment:
I think these should just be called dynamically in the functions, rather than set as global variables

Reply:
I also considered this. I left the global variables because this way you only have to get the options once, instead of every time a function is called. However, I'm not very familiar with how the engine works, and the functions aren't called that often, so you're probably right.

Reply:
No because this would mean if you changed the config during your python session, you would still be using the old value. It definitely needs to be changed.

Reply:
Right, good point 👍